source/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter.h

```diff
@@ -52,6 +52,8 @@ class Http1BridgeFilter : public Http::StreamFilter {
 private:
   void chargeStat(const Http::ResponseHeaderOrTrailerMap& headers);
   void setupStatTracking(const Http::RequestHeaderMap& headers);
+  void doTrailers(const Http::ResponseHeaderOrTrailerMap& trailers,
```

Comment:
Could you please add a link to the spec here that says that trailer-only responses (link) do not need to set grpc status and message?